dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/formModel.vue

```diff
@@ -17,7 +17,10 @@
 <template>
   <div class="form-model-wrapper" v-clickoutside="_handleClose">
     <div class="title-box">
-      <span class="name">{{ $t("Current node settings") }}</span>
+      <span class="name">{{ $t("Current node settings") }}
+        <a v-if="helpUrlEnable(nodeData.taskType)" class="helper-link" target="_blank"
+           :href="helpUrl(nodeData.taskType)">?{{nodeData.taskType}} {{ $t('Instructions') }}</a>
```

Comment:
<i class="el-icon-question" />
{{nodeData.taskType}} {{ $t('Instructions') }}

Reply:
tks, you have provided so beautify icon